bodhi/server/services/graphql.py

```diff
@@ -17,18 +17,56 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """Defines API endpoints related to GraphQL objects."""
 from cornice import Service
+from webob_graphql import serve_graphql_request
+import graphene
+
+from bodhi.server.graphql_schemas import Release
+
 
 graphql = Service(name='graphql', path='/graphql', description='graphql service')
 
 
 @graphql.get()
 def graphql_get(request):
     """
-    Return "Hello World".
+    Perform a GET request.
+
+    Args:
+        request (pyramid.Request): The current request.
+    Returns:
+        The response from Bodhi to the request.
+    """
+    context = {'session': request.session}
+
+    # Setting graphiql_enabled=True
+    return serve_graphql_request(request, schema, graphiql_enabled=True, context_value=context)
+
+
+@graphql.post()
+def graphql_post(request):
+    """
+    Perform a POST request.
 
     Args:
         request (pyramid.Request): The current request.
     Returns:
-        str: A string "Hello World".
+        The response from Bodhi to the request.
     """
-    return "Hello World"
+    context = {'session': request.session}
+
+    # Setting graphiql_enabled=True
+    return serve_graphql_request(request, schema, graphiql_enabled=True, context_value=context)
+
+
+class Query(graphene.ObjectType):
+    """Allow querying objects."""
+
+    allReleases = graphene.List(Release)
+
+    def resolve_releases(self, info):
```

Comment:
Looks like @cverna was right, graphene handles the mapping under the hood for us, so `resolve_allReleases` will work here.